### ISSUE TITLE
Admin menu UI is very annoying

### ISSUE THREAD

**Issue by JimKerslake** [cloudscribe/cloudscribe#992]
The nav linjks in the main admin menu stretch to occupy the full horizontal blank space, so you keep clicking them by accident.

**Comment by CrispinF** [comment]
TO DO: document the CSS change needed in existing projects.
Add the fix so it takes effect in future projects built from template.

**Comment by markidox** [comment]
For some reason this either never made it into the template (which has been updated since this I believe) or has been reversed out?

**Comment by JimKerslake** [comment]
@markidox  yes I guess the ticket was made against the wrong repo really, since the SCSS stuff never gets NuGet-ed and delivered from the main cloudscribe repo... it just gets delivered from the WebApp in the template repo.

I'll patch that into the template repo now - though it may be a while before the next template rebuild.

For existing projects - simple SCSS fix:

```
.nav-link {
    display: inline-block;
    padding: 0.4rem; }

```

(of course remembering that it needs a gulp rebuild.)
